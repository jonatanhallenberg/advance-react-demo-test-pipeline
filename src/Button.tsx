export const Button = () => {
  return <button className="hej" onClick={() => {}}>learn react</button>; 
}
