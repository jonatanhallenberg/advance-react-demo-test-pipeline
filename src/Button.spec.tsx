import { render, screen } from '@testing-library/react';
import { Button } from './Button';
import '@testing-library/jest-dom';

test('renders learn react link', () => {
    const { asFragment } = render(<Button />);
    expect(asFragment()).toMatchSnapshot();
});