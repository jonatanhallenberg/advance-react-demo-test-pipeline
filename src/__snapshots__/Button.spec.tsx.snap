// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <button
    class="hej"
  >
    learn react
  </button>
</DocumentFragment>
`;
